src/pages/LoginRegister/LoginForm.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
```

Comment:
Please replace all react component files to have the file extension .jsx instead of .js. This will help with the readability of the codebase.

Reply:
okay, I'll change that. Thank you for you suggestions.